source/code-snippets/usage-examples/distinct.js

```diff
@@ -7,18 +7,21 @@ const client = new MongoClient(uri);
 
 async function run() {
   try {
-    // define a database and collection on which to run the method
+    
+    // Get the database and collection on which to run the operation
     const database = client.db("sample_mflix");
     const movies = database.collection("movies");
 
-    // specify the document field
+    // Specify the document field to find distinct values for
     const fieldName = "year";
 
-    // specify an optional query document
+    // Specify an optional query document to limit results
```

Comment:
S: just because "Limit" is also a mongodb operation

```suggestion
    // Specify an optional query document to narrow results
```